tests/performance_tests/README.md

```diff
@@ -540,8 +540,8 @@ Performance Test Basic Single Test:
                                         activePrivateKey pairs.
 * `--peer-endpoint arg` (=127.0.0.1)    set the peer endpoint to send
                                         transactions to
-* `--port arg` (=9876)                  set the peer endpoint port to send
-                                        transactions to
+* `--ports arg` (=9876)                 Comma separated list of peer endpoint
+                                        ports to send transactions to
```

Comment:
I believe a single `trx_generator` instance still only sends trxs to a single `host:port`